### DISCUSSION TITLE
Error AlignCenter() use

### DISCUSSION THREAD

**Discussion by QusaiImayyah** [QuestPDF/QuestPDF#436] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/20271178/208066466-6c40294d-3885-46aa-a66f-4b7d62a8e9ec.png)
----------------------------------------------------------------------------------------------------------------------------------------------
![image](https://user-images.githubusercontent.com/20271178/208065608-b8f50b0e-f455-44dd-b107-d28ac89fd369.png)

container.Column(column =>
{
column.Item().ExtendHorizontal().ShowOnce().PaddingBottom(5).DebugArea().AlignCenter().Text("التقرير التجريبي").Style(Typography.Title);
});
or 
container.Column(column =>
{
column.Item().Extend().ShowOnce().PaddingBottom(5).DebugArea().AlignCenter().Text("التقرير التجريبي").Style(Typography.Title);
});
--------------------------------
error Image ExtendHorizontal Height:
column.Item().ExtendHorizontal().Image(Logo, ImageScaling.FitArea); ------ Large Image
---                                
column.Item().ExtendHorizontal().Height(100).Image(Logo, ImageScaling.FitArea); ------- small 
-----
![image](https://user-images.githubusercontent.com/20271178/208066267-c0edda73-103b-43cb-a73d-d95c4a2a7bd7.png)

**Comment by girlpunk** [comment, marked as answer]
```csharp
column.Item()
    .ExtendHorizontal()
    .Row(row =>
    {
        row.AutoItem().AlignMiddle().Height(100).Image(Logo, ImageScaling.FitArea);
    });
```

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
